src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -69,6 +70,8 @@ def handle_exception(ex):  # pylint: disable=too-many-return-statements
     from azure.common import AzureException
     from azure.core.exceptions import AzureError
 
+    log_exception_history(cli_ctx, ex)
```

Comment:
You can get more exception info if you put this `log_exception_history(cli_ctx, az_error)` to line 145.